includes/Admin/Admin_AJAX.php

```diff
@@ -46,6 +50,100 @@ public function get_nonce() {
 		return wp_create_nonce( self::NONCE_KEY );
 	}
 
+	/**
+	 * Handles the AJAX request to setup the runtime environment if needed.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function setup_environment() {
```

Comment:
Nit-pick: "Setup" is a noun, we should use the verb "set up". This also applies to the AJAX action etc.
```suggestion
	public function set_up_environment() {
```

Reply:
Thanks, updated